backend.py

```diff
@@ -88,20 +89,19 @@ def get_board(data):
     """
     paths = get_paths(data)
     coordinates = get_coordinates(data)
+    board = {coordinate: [] for coordinate in coordinates}
     rotation_dict = {0: 0, 10: 90, 12: 180, 6: 270}
-    board = {}
     for layer in data['layers']:
-        tiles_layer = []
-        for data in layer['data']:
+        for ind in range(len(layer['data'])):
+            data = layer['data'][ind]
```

Comment:
Místo:
```
for i in range(len(X)):
    data1 = X[i]
    data2 = Y[i]
    ...
```
je téměř vždy lepší použít:
```
for data1, data2 in zip(X, Y):
    ...
```

Reply:
Vím, že to není nejlepší řešení a ani se mi nelíbí. Problém je v tom, že pomocí toho indexu procházím dvě proměnné - data v JSONu a vznikající slovník. Momentálně je to řešení, které mi funguje, ale zkusím to ještě promyslet.

Reply:
S tím jsem včera hodně bojovala. Díky za postrčení.